src/main/java/org/fhirvp/notification/impl/NotificationHandlerImpl.java

```diff
@@ -0,0 +1,19 @@
+package org.fhirvp.notification.impl;
+
+import io.quarkus.arc.Unremovable;
+import org.fhirvp.notification.FHIRNotificationEvent;
+import org.fhirvp.notification.NotificationHandler;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+@Unremovable
```

Comment:
We should add a comment why this is needed.

Reply:
Addressed at 1f439b8